Libraries/LibWeb/CredentialManagement/Credential.cpp

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (c) 2025, Altomani Gianluca <altomanigianluca@gmail.com>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <LibWeb/Bindings/Intrinsics.h>
+#include <LibWeb/CredentialManagement/Credential.h>
+
+namespace Web::CredentialManagement {
+
+JS::ThrowCompletionOr<GC::Ref<WebIDL::Promise>> Credential::is_conditional_mediation_available(JS::VM& vm)
```

Comment:
should have a spec link here in commit #3 where the function is added. even if the impl is "throw lmao"

Reply:
Added them everywhere